.github/actions/run-e2e/action.yml

```diff
@@ -37,6 +37,12 @@ runs:
           echo "APP_CONFIG_OAUTH2_REDIRECT_LOGOUT=/"
           echo "APP_CONFIG_OAUTH2_REDIRECT_LOGIN=/"
           echo "APP_CONFIG_OAUTH2_REDIRECT_SILENT_IFRAME_URI={protocol}//{hostname}{:port}/assets/silent-refresh.html"
+          echo "APP_CONFIG_ENABLE_DOWNLOAD_PROMPT=true"
```

Comment:
update main readme file with these variables

Reply:
Updated the README.md file. The values I added to that file are -

APP_CONFIG_ENABLE_DOWNLOAD_PROMPT=true
APP_CONFIG_ENABLE_DOWNLOAD_PROMPT_REMINDERS=true
APP_CONFIG_DOWNLOAD_PROMPT_DELAY=`<time>`
APP_CONFIG_DOWNLOAD_PROMPT_REMINDER_DELAY=`<time>`
APP_CONFIG_ENABLE_FILE_AUTO_DOWNLOAD=true
APP_CONFIG_FILE_AUTO_DOWNLOAD_SIZE_THRESHOLD_IN_MB=`<file-size>`

Please validate if these are correct or if we should add the actual values instead.